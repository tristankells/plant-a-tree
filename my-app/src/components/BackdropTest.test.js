import React from "react";
import { shallow } from "enzyme";

import Backdrop from "./Backdrop";

describe("Backdrop", () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Backdrop debug />);

    expect(component).toMatchSnapshot();
  });
});
