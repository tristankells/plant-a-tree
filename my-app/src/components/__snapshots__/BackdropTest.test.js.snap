// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop should render correctly in "debug" mode 1`] = `
<div
  className="hide"
  id="backdrop"
/>
`;
